src/main/resources/static/js/reservation.js

```
@@ -4,176 +4,176 @@ const TIME_API_ENDPOINT = '/times';
+  input.className = 'form-control';
+  return input;
 }
 
 function createActionButton(label, className, eventListener) {
-    const button = document.createElement('button');
-    button.textContent = label;
-    button.classList.add('btn', className, 'mr-2');
-    button.addEventListener('click', eventListener);
-    return button;
+  const button = document.createElement('button');
+  button.textContent = label;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createActionButton'. Consider to delete it or rename it.

'createActionButton' is defined here:


(lint/suspicious/noRedeclare)